packages/manfred2word/src/engine/doc-parts/language-section/language-section.part.ts

```diff
@@ -0,0 +1,53 @@
+import { ManfredAwesomicCV } from '@/model';
+import { LanguageVm } from './language-section.vm';
+import { TableCell, TableRow, Table } from 'docx';
+import { titleLanguageSection, sectionLanguageSection } from './sections-languages-section-parts';
+import { styles } from './language-section.styles';
+import { mapFromCvToLanguageVm } from './language-section.mapper';
+
+export const generateLanguageSection = (cv: ManfredAwesomicCV): Table => {
+  const profileSectionVm = mapFromCvToLanguageVm(cv);
+
+  return generateLanguageSectionInner(profileSectionVm);
+};
+
+const generateLanguageSectionInner = (languageSectionVm: LanguageVm[]): Table => {
+  if (Array.isArray(languageSectionVm) && languageSectionVm.length >= 1) {
+    return new Table({
+      ...styles.table,
+      rows: generateSectionLanguageFromVmToRows(languageSectionVm),
+    });
+  } else {
+    return new Table({
+      rows: [],
+    });
+  }
+};
+
+export const generateSectionLanguageFromVmToRows = (sectionLanguageVm: Array<LanguageVm>) => {
+  let result = [];
+
+  result = sectionLanguageVm.map(
+    (language: LanguageVm) =>
+      new TableRow({
+        children: [
+          new TableCell({
+            ...styles.table,
+            children: [sectionLanguageSection(language)],
+          }),
+        ],
+      })
+  );
+
+  result.unshift(
+    new TableRow({
+      children: [
+        new TableCell({
+          children: [titleLanguageSection()],
+        }),
+      ],
+    })
+  );
```

Comment:
I think this is not necessary because if the languages do not exist or the array is empty, the title will not be displayed. Can you check it?